drivers/sensor/bma4xx/bma4xx.c

```diff
@@ -0,0 +1,687 @@
+/* Bosch BMA4xx 3-axis accelerometer driver
+ *
+ * Copyright (c) 2023 Google LLC
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT bosch_bma4xx
+
+#include <zephyr/init.h>
+#include <zephyr/sys/byteorder.h>
+#include <zephyr/sys/__assert.h>
+#include <zephyr/logging/log.h>
+#include <zephyr/pm/device.h>
+
+LOG_MODULE_REGISTER(bma4xx, CONFIG_SENSOR_LOG_LEVEL);
+#include "bma4xx.h"
+
+#ifdef CONFIG_PM_DEVICE
+static int bma4xx_pm_action(const struct device *dev, enum pm_device_action action)
+{
+	return -ENOTSUP;
+}
+#endif /* CONFIG_PM_DEVICE */
```

Comment:
Your call, we don't need power management so I'm fine with this just being all NULL.